### DISCUSSION TITLE
Export Playlist Files (mp3, flac, ect.) to Folder

### DISCUSSION THREAD

**Discussion by 1tituz** [Taiko2k/Tauon#978] [Ideas]
First i´de like to thank you for this Music Player, it's exactly what i was looking for after using Winamp for decades!

Now there was one feature in Winamp (a plugin i think) where you could export the files of a playlist to a folder.
That was for my usecases quite handy sometimes, since i transfer playlists to other devices.

Do you think this feature would be possible to add to the 'right click on Playlist / Export...' Menu?
  
![screen](https://user-images.githubusercontent.com/101503597/234548261-f8b5ff18-319c-4f43-9c35-07f7ce424324.png)

Anyways, keep it up,
Thanks!

**Comment by Taiko2k** [comment]
There is the sync feature, I wonder if that would work for what you're looking for.

![Screenshot from 2023-04-27 01-05-52](https://user-images.githubusercontent.com/17271572/234584220-ed700805-b4ea-42dc-b000-0e0161a0940c.png)

You select the playlist by right click playlist > Misc > Set as sync playlist. Then if you want, untick transcode files. Then specify the folder you want to output to. Then click Start Sync.

Though note the output will be flattened into albums. Maybe that's what you want though.

**Reply by 1tituz** [reply]
Thanks for the quick answer!
Unfortunatly i only want those Songs i dropped into that Playlist, not the whole Album of those Songs.

At the moment i use the exported "playlist.xspf" file and a Batch script to export those Songs. That works for me, i just thought it would be a nice thing to have. And it would be easier for everybody, if there is demand for it?!
